### PR TITLE
Update wave worker to support recorder.js

### PR DESCRIPTION
**Pull request by ForTheYin**
Very minor changes to get `waveWorker.js` working with `Recorder.js`.

The big commit diff comes from the whitespace changes (to match with `encoderWorker.js`). See https://github.com/chris-rudmin/Recorderjs/pull/88/files?w=1 for better visibility.

### CODE REVIEW

**Comment by chris-rudmin** [comment]
This is great!  Honestly, I hadn't considered using the wave worker that way.

File: src/waveWorker.js [hunk 1]
```diff
@@ -1,119 +1,124 @@
...
-
-        default:
-          throw "Only 8, 16, 24 and 32 bits per sample are supported";
+      default:
+        // Ignore any unknown commands and continue recieving commands
+    }
+  };
+
+  var WavePCM = function( config ){
+    this.sampleRate = config['encoderSampleRate'] || 44100;
```
Comment (chris-rudmin) [inline src/waveWorker.js]: I feel that using `encoder...` in the config naming is incorrect, as in a riff WAV file there is no encoding going on.       Perhaps we could use `wavSampleRate` and `wavBitDepth` and specify in the readme that these are the only config options that work with wav.

File: src/waveWorker.js [hunk 2]
```diff
@@ -1,119 +1,124 @@
...
+    this.bitDepth = config['encoderBitDepth'] || 16;
```
Comment (chris-rudmin) [inline src/waveWorker.js]: This config option is missing from the Readme.

File: src/waveWorker.js (hunk 1 above)
Comment (chris-rudmin) [inline src/waveWorker.js]: Also if you have the time, I would love it if we could add unit tests for these config options.

**Comment by ForTheYin** [comment]
@chris-rudmin Thanks for reviewing my code!

I updated the config naming to use a `wav` prefix instead of `encoder`, and added some unit tests.

**Comment by chris-rudmin** [comment]
@ForTheYin Thanks for the great contribution. I will merge this and make a few more changes in addition!

**Pull request merged**